spec/factories/claims.rb

```diff
@@ -116,12 +128,25 @@
       work_items { [build(:work_item, :valid)] }
     end
 
+    trait :uplifted_work_item do
+      work_items { [build(:work_item, :with_uplift)] }
+    end
```

Comment:
so you could change this one to:
```ruby
    trait :uplifted_work_item do
      with_enhanced_rates
      work_items { [build(:work_item, :with_uplift)] }
    end
```
this would help ensure data is valid -> can only have uplift when enhanced rate

Reply:
Tried to be as minimal as possible with what i was adding but i understand and have updated.